### DISCUSSION TITLE
meta-tegra for X86_64 target

### DISCUSSION THREAD

**Discussion by ebail** [OE4T/meta-tegra#525] [General]
Hi,

I had good experience working with meta-tegra on Jetson TX2 target.
Good Yocto integration, nice job.

I now want to play with Cuda and its related packages on X86_64 target with NVidia GPU.

meta-tegra has not compatible machine with that. I will override that. I am wondering if someone already test that and what is their conclusion ? An other option would be to use container to run NVidia available images (e.g Ubuntu).
Any other alternative ?

Thanks,

Eloi

**Comment by madisongh** [comment]
This topic came up a few years ago (see #23). It should be possible to adapt the CUDA support here for use with x86 systems and discrete GPUs.
